src/main/java/spoon/Launcher.java

```diff
@@ -88,6 +90,7 @@
 	protected JSAPResult jsapActualArgs;
 
 	private List<String> processors = new ArrayList<String>();
+	private List<Processor<? extends CtElement>> processorsInstance = new ArrayList<Processor<? extends CtElement>>();
```

Comment:
I do NOT like the name. Why not just call it "processors" without the word "Instance"?